dags/oss_know/libs/email_address/init_email_address_from_ck.py

```diff
@@ -0,0 +1,179 @@
+import datetime
+
+from oss_know.libs.util.clickhouse_driver import CKServer
+from oss_know.libs.util.base import infer_country_from_emailcctld, infer_country_from_emaildomain, \
+    infer_company_from_emaildomain, infer_country_from_company
+
+
+def load_all_email_address(clickhouse_server_info):
+    ck = CKServer(host=clickhouse_server_info["HOST"],
+                  port=clickhouse_server_info["PORT"],
+                  user=clickhouse_server_info["USER"],
+                  password=clickhouse_server_info["PASSWD"],
+                  database=clickhouse_server_info["DATABASE"])
+
+    gits_sql = "SELECT DISTINCT author_email,committer_email FROM gits"
+    gits_email_from_ck = ck.execute_no_params(gits_sql)
+    all_email_address_dict = {}
+    for author_email, committer_email in gits_email_from_ck:
```

Comment:
A possible better solution: get two types of emails with DISTINCT from the table. Make them 2 sets, then get the union